pinot-common/src/main/java/org/apache/pinot/common/metrics/MinionMeter.java

```diff
@@ -38,7 +38,8 @@ public enum MinionMeter implements AbstractMetrics.Meter {
   SEGMENT_BYTES_UPLOADED("bytes", false),
   RECORDS_PROCESSED_COUNT("rows", false),
   RECORDS_PURGED_COUNT("rows", false),
-  COMPACTED_RECORDS_COUNT("rows", false);
+  COMPACTED_RECORDS_COUNT("rows", false),
+  GROOVY_SECURITY_VIOLATIONS("exceptions", true);
```

Comment:
Let's clean up the unrelated changes. The metric should be emitted from controller for table config, and broker for query

Reply:
I have removed the metric part.